admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/SchemaServiceTest.java

```diff
@@ -160,15 +160,15 @@ public void getAllDefinitionsTest() throws JSONException {
 	
 	@Test
 	@WithUserDetails("reg-officer")
-	public void testFetchAllDynamicFields() throws Exception {		
+	public void testFetchAllDynamicFields() {
```

Comment:
please follow the naming convention for test name

Reply:
Gave the correct naming convention for the testMethod